Content.Client/Construction/UI/ConstructionMenuPresenter.cs

```diff
@@ -166,10 +166,23 @@ private void OnGridViewRecipeSelected(object? sender, ConstructionPrototype? rec
             if (_placementManager.IsActive && !_placementManager.Eraser) UpdateGhostPlacement();
             PopulateInfo(_selected);
         }
-
-        private void OnViewPopulateRecipes(object? sender, (string search, string catagory) args)
+		
+		private int checkfuzzysearch(string hostfield,string searchtext){// return an int for futureproofing, if you wanted to sort by likeness, or something. doesn't matter much now, ints are compatible with boolean logic, anyways.
```

Comment:
CheckFuzzySearch, also this is not a docstring

Reply:
fixed in commit